includes/admin/class-wcmp-export.php

```diff
@@ -335,7 +335,9 @@ public function add_shipments(array $order_ids, bool $process)
             $extraOptions = WCX_Order::get_meta($order, WCMYPA_Admin::META_SHIPMENT_OPTIONS_EXTRA);
             $colloAmount  = $extraOptions["collo_amount"] ?? 1;
 
-            if ($colloAmount > 1) {
+            $isLocalShipment = ('NL' === $order->get_shipping_country());
```

Comment:
Ik zou van de `NL` een constante gebruiken -> deze hebben we volgens mij beschikbaar in de SDK